ngraph/src/ngraph/op/non_max_suppression.cpp

```diff
@@ -291,7 +290,7 @@ bool ngraph::op::v3::NonMaxSuppression::visit_attributes(AttributeVisitor& visit
     return true;
 }
 
-void op::v3::NonMaxSuppression::validate_and_infer_types()
+void op::v3::NonMaxSuppression::validate()
```

Comment:
rename this method to smth more meaningful like validate_inputs

Reply:
The original method name was 'validate_inputs' but I got a comment that the method also validates the attributes. So I renamed it to 'validate'. I guess that the name 'validate' is a good one since it is generic and used in other projects.